docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,382 @@
+Android project configured to use gradle, your application code, and a support
+package containing the Python interpreter and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take quite a few minutes the first time, depending on
+your Internet connection speed, and then should take less than 30 seconds every
+successive time. If you see that Gradle is "CONFIGURING" for many, many minutes,
+this is normal; Gradle is busy performing downloads, and you should not
+interrupt it.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase build android
+  [helloworld] Building Android APK...
+  Starting a Gradle Daemon
+  ...
+  BUILD SUCCESSFUL in 1m 1s
+  28 actionable tasks: 17 executed, 11 up-to-date
+  [helloworld] Built android/Hello World/app/build/outputs/apk/debug/app-debug.apk
+
+Run the app on a virtual device
+===============================
+
+We're now ready to run our application. You can use Briefcase's ``run`` command
+to run the app on an Android device, either physical or virtual. This section
+covers running the app on a virtual Android device.
+
+On Android, Briefcase's ``run`` command requires that you specify which device to run on.
+If you use ``run`` without specifying a device, Briefcase will tell you how to
+create an appropriate Android virtual device. Run that command on your system.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      No Android device was specified. Please specify a specific device on which
+      to run the app by passing `-d <device_id>`.
+
+      ...
+
+      If you do not see any devices, you can create and start an emulator by running:
+
+      ...
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run android
+
+      No Android device was specified. Please specify a specific device on which
+      to run the app by passing `-d <device_id>`.
+
+      ...
+
+      If you do not see any devices, you can create and start an emulator by running:
+
+      ...
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>briefcase run android
+
+      No Android device was specified. Please specify a specific device on which
+      to run the app by passing `-d <device_id>`.
+
+      ...
+
+      If you do not see any devices, you can create and start an emulator by running:
+
+      ...
+
+
+Examine the specific output on your system, and run the commands provided
+until you see an Android virtual device on your screen. It will look something
+like this as it starts up.
+
+.. figure:: ../images/android/tutorial-5-booting.png
+   :alt: Android virtual device booting
+
+   Android virtual device booting
+
+.. figure:: ../images/android/tutorial-5-running.png
```

Comment:
Worth adding "once the emulator has started, it will show you an Android launchpad screen' (or similar text) to clarify why there's two screenshots.